jaggery-modules/carbon-metrics/module/scripts/timers/timers.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+(function(metrics) {
+    var MetricManager = org.wso2.carbon.metrics.manager.MetricManager;
+    var Level = org.wso2.carbon.metrics.manager.Level;
+    var pushContext = function(timer) {
+        if (typeof WSO2_METRICS_TIMERS === 'undefined') {
+            WSO2_METRICS_TIMERS = [];
+        }
+        WSO2_METRICS_TIMERS.push(timer);
+    };
+    var popContext = function() {
+        if (typeof WSO2_METRICS_TIMERS === 'undefined') {
+            return null; //Do nothing since there are no timers
+        }
+        return WSO2_METRICS_TIMERS.pop();
+    };
+    var getlastInvocation = function() {
+        if (typeof(WSO2_METRICS_LAST_INVOCATION) === 'undefined') {
```

Comment:
Thanks for mentioning , As I know it is a good practice in JavaScript to use === rather than strict comparison.
Source: http://www.w3schools.com/js/js_best_practices.asp